cli/enginecli/test_dropper.py

```diff
@@ -1137,36 +1151,106 @@ def test_claim_erc721_with_custom_amount(self):
         )
 
         current_block = len(chain)
-        block_deadline = current_block  # since blocks are 0-indexed
+        block_deadline_0 = current_block  # since blocks are 0-indexed
 
-        request_id = 89390589485
-        message_hash = self.dropper.claim_message_hash(
-            claim_id, request_id, accounts[1].address, block_deadline, custom_amount
-        )
-        signed_message = sign_message(message_hash, self.signer_0)
+        request_id_0 = 89390589485
 
-        self.assertEqual(self.nft_contract.owner_of(token_id), self.dropper.address)
-        self.assertFalse(self.dropper.claim_status(claim_id, request_id))
+        for idx, token_id_as_ammount in enumerate(token_ids):
+            request_id = request_id_0 + idx
+            block_deadline = block_deadline_0 + idx
+            message_hash = self.dropper.claim_message_hash(
+                claim_id,
+                request_id,
+                accounts[1].address,
+                block_deadline,
+                token_id_as_ammount,
+            )
+            signed_message = sign_message(message_hash, self.signer_0)
 
-        self.dropper.claim(
-            claim_id,
-            request_id,
-            block_deadline,
-            custom_amount,
-            self.signer_0,
-            signed_message,
-            {"from": accounts[1]},
+            self.assertEqual(
+                self.nft_contract.owner_of(token_id_as_ammount), self.dropper.address
+            )
+            self.assertFalse(self.dropper.claim_status(claim_id, request_id))
+
+            self.dropper.claim(
+                claim_id,
+                request_id,
+                block_deadline,
+                token_id_as_ammount,
+                self.signer_0,
+                signed_message,
+                {"from": accounts[1]},
+            )
+            self.assertEqual(
+                self.nft_contract.owner_of(token_id_as_ammount), accounts[1].address
+            )
+            self.assertTrue(self.dropper.claim_status(claim_id, request_id))
+
+    def test_claim_erc721_multiple_claimants(self):
+        token_ids = [107, 108, 109]
```

Comment:
Make a note that this list should be small (less than 8 tokens, I believe) because we iterate over the auto-generated accounts in the account list.

Just a comment in case ganache default behaviour ever reduces the number of autogenerated accounts.